artifactory/commands/config.go

```diff
@@ -415,3 +415,15 @@ type ConfigCommandConfiguration struct {
 	Interactive bool
 	EncPassword bool
 }
+
+func GetAllServerIds() []string {
```

Comment:
I suggest moving this function to **artifactory/utils/config/config.go** and renaming it to *GetAllArtifactoryServerIds*.